templates/semantic-ui/invenio_communities/frontpage.html

```diff
@@ -0,0 +1,92 @@
+{# -*- coding: utf-8 -*-
+
+  This file is part of Invenio.
+  Copyright (C) 2016-2020 CERN.
+
+  Invenio is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+{% extends "invenio_communities/base.html" %}
+
+{%- block javascript %}
+  {{ super() }}
+  {{ webpack['invenio-communities-frontpage.js'] }}
+{%- endblock %}
+
+{%- block page_body %}
+  <div class="ui container fluid page-subheader-outer ml-0-mobile mr-0-mobile">
+    <div class="ui container centered grid page-subheader rel-pt-1">
+      <div class="one column row">
+        <div class="eight wide center aligned column mt-auto mb-auto">
+          <h1 class="ui header mr-10">{{ _('Communities') }}</h1>
+          <p class="text-muted">Created and curated by Zenodo users. 
+            {% if not current_user.is_authenticated %}
+              <a href="#community-info">Read more</a>
+            {% endif %}
+          </p>
+        </div>
+      </div>
+      <div class="two column stackable row">
+        <div class="eight wide tablet five wide computer column mt-auto mb-auto">
+          <form action="{{ url_for('invenio_communities.communities_search') }}" class="ui form">
+            <div class="ui fluid action input">
+              <input type="text" name="q" class="form-control" placeholder="{{ _('Search communities') }}">
+              <button type="submit" class="ui icon search button"><i class="search icon"></i></button>
+            </div>
+          </form>
+        </div>
+        <div class="three wide mobile five wide tablet three wide computer stretched column">
+          <a href="{{ config.COMMUNITIES_ROUTES['new'] }}" class="ui icon left labeled positive button" role="button">
+            <i class="icon plus"></i>
+            {{_('New community')}}
+          </a>
+        </div>
+      </div>
+      <div class="ui divider hidden"></div>
+    </div>
+  </div>
+  <!-- TODO: Add Featured communities slideshow here -->
+  <div class="ui container communities-frontpage rel-mt-3 rel-mb-2">
+    <div class="flex align-items-center">
+      <h2 class="ui header mb-0">{{ _('New communities') }}</h2>
+      <a class="rel-ml-1" href="{{url_for('invenio_communities.communities_search')}}">
+        {{ _('See all')}}
+      </a>
+    </div>
+    <div class="ui divider hidden"></div>
+    <div id="featured-communities" class="communities-list"></div>
+  </div>
+
+  <div class="ui divider"></div>
+  <div class="ui fluid container bg-gray">
+    <div class="ui container rel-pt-3">
+      <div class="ui stackable grid" id="community-info">
+        <!-- TODO: Add information about communities here -->
```

Comment:
leftovers?

Reply:
No, we don't have this content yet, so I left it commented (see my PR message)